desc/grid.py

```diff
@@ -441,9 +550,10 @@ def __init__(self, nodes, sort=False, jitable=False):
             ) = self._find_unique_inverse_nodes()
             self._weights = self._scale_weights()
 
-        self._L = self.num_rho
-        self._M = self.num_theta
-        self._N = self.num_zeta
+        self._L = self.num_nodes
+        self._M = self.num_nodes
+        self._N = self.num_nodes
```

Comment:
Might be better to set it to 0 in case users think its a resolution. For example LinearGrid(L, M, N) would we very large if each were grid.num_nodes.

Reply:
This is probably fine, it is actually pretty rare that we use `grid.L` etc., and even rarer that we use it in order to make another grid